src/includes/getting-started-install/android.mdx

```diff
@@ -1,4 +1,34 @@
-To install the Android SDK, add it your `build.gradle` file:
+### Auto-Installation With the Sentry Android Gradle Plugin
```

Comment:
Worth adding a note that this is only for >= 3.1.0

Reply:
True. Actually am not sure should we add it here since it's gonna be a preview cc @bruno-garcia ?